app/services/instagram/message_text.rb

```diff
@@ -56,6 +57,10 @@ def contacts_first_message?(ig_scope_id)
     @inbox.contact_inboxes.where(source_id: ig_scope_id).empty? && @inbox.channel.instagram_id.present?
   end
 
+  def sent_via_test_webhook?
+    @messaging[:sender][:id] == '12334' && @messaging[:recipient][:id] == '23245'
```

Comment:
@tejaswinichile, do we have documentation on Facebook about these test ids? They seem pretty random. What are the chances that it will be changed in future?

Reply:
They have changed the whole test webhook JSON altogether on the recent release, that's the test webhook data wasn't getting populated. 
But for the current META version, it is constant.

<img width="479" alt="Screenshot 2022-08-22 at 1 49 52 PM" src="https://user-images.githubusercontent.com/10827585/185874158-1aad4414-81ab-4cc4-b7ce-c47937fe1423.png">